.github/workflows/publish-to-edge.yaml

```diff
@@ -0,0 +1,33 @@
+name: Publish to edge
```

Comment:
I think we could rename this to `Test and publish to edge` or something like that.  

Also, at present, the testing will execute twice on push events (once in this workflow, and once because `ci.yaml` triggers on pushes).  I think we can also make `ci.yaml` only test on `pull_request` events, and then `push` event testing would be covered here.  Thoughts?

Reply:
I agree. `push` part removed from `ci.yaml`, this will be covered by the reusable worklfow